### DISCUSSION TITLE
[ERROR] No suitable model found for J1415+1112. It may be that models are older than the universe or that the χ² are very large.

### DISCUSSION THREAD

**Discussion by Astro-JiangDong** [mboquien/cigale#66] [Q&A] [answered]
Hello, everyone!

I am currently attempting to fit the spectral energy distribution (SED) of a late-type galaxy (its redshift is 0.3152). I have tried various combinations of parameters, but they have all produced the same result: 'No suitable model found.' Could you please provide some advice on parameter selection?

The following are the pcigale file and input data.

[pcigale.txt](https://github.com/mboquien/cigale/files/12576495/pcigale.txt)
[input_data.txt](https://github.com/mboquien/cigale/files/12576497/input_data.txt)

The following are the response curve data for the seven bands I have used.

[DECam_z.txt](https://github.com/mboquien/cigale/files/12576539/DECam_z.txt)
[DECam_r.txt](https://github.com/mboquien/cigale/files/12576540/DECam_r.txt)
[PS1_i.txt](https://github.com/mboquien/cigale/files/12576541/PS1_i.txt)
[LRIS_H.txt](https://github.com/mboquien/cigale/files/12576544/LRIS_H.txt)
[LRIS_J.txt](https://github.com/mboquien/cigale/files/12576545/LRIS_J.txt)
[LRIS_Ks.txt](https://github.com/mboquien/cigale/files/12576547/LRIS_Ks.txt)
[LRIS_Y.txt](https://github.com/mboquien/cigale/files/12576548/LRIS_Y.txt)

**Comment by mboquien** [comment]
Hello. Looking at it I think the flux uncertainties are much too small. The overall shape of the SED  suggests that there is an issue with the photometry, with large jumps from one band to another, in particular in LRIS bands. When that happens with very small uncertainties, the likelihood underflows to 0, eliminating all the models.

**Reply by Astro-JiangDong** [reply]
I'm sorry, there was indeed an issue with the input data. Below is the corrected data. After multiple attempts, it still yields the same result: 'No suitable model found.' 

I'm fitting the SED of early-type galaxy, not late-type galaxy. I'm sorry to previous mistake.

[input_data_new.txt](https://github.com/mboquien/cigale/files/12608034/input_data_new.txt)

**Reply by mboquien** [reply, marked as answer]
I looked a bit more into it. The issue is in your Y filter. It has a line `col1 col2`, which breaks the import. If you delete that line and import it again, it should work.

**Reply by Astro-JiangDong** [reply]
You're right !!! After I delete "col1" and "col2",   it immediately gave the fitting result (I am too careless). Thank you very much!
